### DISCUSSION TITLE
Does DaisyUI include everything in production builds, or only actually used components?

### DISCUSSION THREAD

**Discussion by kevinrenskers** [saadeghi/daisyui#2230] [Q&A] [answered]
When I include DaisyUI as a TailwindCSS plugin, does that mean that only the components that I actually use will become part of my build? Tailwind itself specifically says this in their docs:

> Tailwind CSS is incredibly performance focused and aims to produce the smallest CSS file possible by only generating the CSS you are actually using in your project.

But I couldn't find anything like this in the DaisyUI docs. So I am wondering if including DaisyUI will include all components, also the ones I won't be using, or if it will only include the things that I actually use.

**Comment by saadeghi** [comment, marked as answer]
No, it won't include all components.  
It works just like Tailwind CSS class names.  
Only the class names you use will be included in production.

**Reply by kevinrenskers** [reply]
Awesome, thanks for the quick answer!
